File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/gettingstarted/reverse_a_number.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    while(n>0){
        int x=n%10;
        n=n/10;
        cout<<x<<"
";
    }
    return 0;
    
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/ReverseANumber.java
import java.util.*;
   
   public class Main{
   
   public static void main(String[] args) {
     // write your code here
     Scanner sc=new Scanner(System.in);
     int n=sc.nextInt();
     String s=Integer.toString(n);
     for(int x=s.length()-1;x>=0;x--){
         System.out.println(s.charAt(x));
     }
    }
   }